build/aclocal/bakefile.m4

```diff
@@ -76,7 +76,7 @@ AC_DEFUN([AC_BAKEFILE_PLATFORM],
 
     if test "x$BAKEFILE_FORCE_PLATFORM" = "x"; then
         case "${BAKEFILE_HOST}" in
-            *-*-mingw32* )
+            *-*-mingw* )
```

Comment:
Is this intentional? If so, why?

Reply:
Yes, it extends the accepted values to both `*-*-mingw32*` and `*-*-mingw64*`. The `gcc` version that came with `Qt` identified itself as the latter.
Alternatively the following should work just as well: `*-*-mingw32* | *-*-mingw64* )`.

Reply:
I was just wondering how did it always work for us with MinGW64 before, without this change.

What exactly do you mean by "identified"? Is this part of the triplet returned by `config.guess` or something else?

Anyhow, I'm not really against this change, it makes sense. But I just wish I understood why hadn't it been necessary before. And, as usual, it would be better if it could be done in a separate commit.

TIA!

Reply:
No idea about MinGW64.
In my config log there are:
```
configure:3699: checking build system type
configure:3713: result: x86_64-pc-mingw64
configure:3733: checking host system type
configure:3746: result: x86_64-pc-mingw64
```
I don't know if that is given by the compiler or the bash environment but one of those is what seems to be used. Anyway, that change was just a lucky guess.
Later in the same log there is
`gcc.exe (i686-posix-dwarf-rev0, Built by MinGW-W64 project) 5.3.0`.

Reply:
The output in `config.log` is from `config.guess`, yes...